src/en/getting-started-jaas.md

```diff
@@ -16,94 +17,54 @@ when registering which is the username that will be used by JAAS.
 
 ## Obtain cloud credentials
 
-In order for JAAS to deploy your workload on the cloud of your choice, you must
-enter credentials for each cloud you would like to use.
+JAAS supports the following public clouds:
+* [Amazon Web Services (AWS)][aws]
+* [Google Compute Engine (GCE)][gce]
+* [Microsoft Azure][azure]
 
 We recommend that you use each public cloud's identity and access management
 (IAM) tool to generate a new set of credentials exclusively for use with JAAS.
 See [Cloud credentials][credentials] for more information. 
 
-## Use JAAS from the web GUI
+## Log in to JAAS
 
-It is easiest to do the initial steps using the web. Open the [JAAS login page][jaaslogin]
-to begin.
+Open the [JAAS login page][jaaslogin] to begin.
 
 Log in to JAAS using your Ubuntu SSO account. This will open a second tab in
 your web browser to log you in. If the tab does not open, check and disable any
 pop-up blockers that may be running and try again.
 
-Press the green + button on the page to design and deploy your model. Press
-Deploy Changes when you are ready.
+## Create and deploy a model
 
-![juju_jaas_main](./media/juju_jaas_main.png)
-
-The last step in the deployment requires and will request your credentials for
-the public cloud you want to use, so prepare these beforehand and have them
-ready.
-
-## Use JAAS from the CLI
-
-To use JAAS from the CLI requires a few additional steps.
-
-### Install Juju
-
-Follow [these instructions][installjuju] to install Juju on your local machine.
-
-### Register or login to JAAS from Juju
-
-If you are using Juju 2.1, add the JAAS controller with:
-
-```bash
-juju register jimm.jujucharms.com
-```
+Press the green + button on the page to add a charm or bundle to your model.
```

Comment:
Let's leave out the charm/bundle and just go with "to start building your model".